challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js

```diff
@@ -0,0 +1,25 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+checkEven();
+console.log("\n");
+checkOdd();
+
+function checkEven() {
+  for (let i = 10; i <= 40; i++) {
```

Comment:
Take care of naming.

```suggestion
  for (let index = 10; index <= 40; index++) {
```